files/root/grafana_tools/bulk_alert/config

```diff
@@ -0,0 +1,2 @@
+user=  ## Grafana admin user
```

Comment:
Suggest: add a comment that this file is managed by Puppet (local changes will be lost).

Reply:
Again, these files come from another git repo. I don't want to start editing or recreating the files.
But a great idea for a future improvement.

Reply:
I'm clearly confused how this content get's into the config ... but I trust you've tested and the item I brought up is not addressable here. Resolving this thread.